.github/cloud-samples-tools/cmd/main.go

```diff
@@ -72,7 +72,7 @@ func affectedCmd(configFile string, diffsFile string) {
 	if err != nil {
 		log.Fatalln("❌ error getting the diffs: ", diffsFile, "\n", err)
 	}
-	diffs := strings.Split(string(diffsBytes), "\n")
+	diffs := strings.Split(strings.TrimSpace(string(diffsBytes)), "\n")
```

Comment:
The tool is broken into the main command which does all the input validation, and the config and utils libraries which expect the inputs in the correct format. The unit tests cover the libraries.

The main command input validation is trickier to test since it involves files, and is simple enough that it doesn't seem worth the effort.